aeron-client/src/main/java/io/aeron/ChannelUriStringBuilder.java

```diff
@@ -80,6 +80,8 @@ public final class ChannelUriStringBuilder
     private Long untetheredWindowLimitTimeoutNs;
     private Long untetheredRestingTimeoutNs;
     private boolean isSessionIdTagged;
+    private Integer streamId;
+    private Integer publicationWindowLength;
```

Comment:
This is already implemented on master. Please merge those into your PR or rebase it.

Reply:
Done